sled-agent/src/bootstrap/params.rs

```diff
@@ -495,6 +503,7 @@ mod tests {
                 bgp: Vec::new(),
                 bfd: Vec::new(),
             },
+            allowed_source_ips: Default::default(),
```

Comment:
I missed where we implemented default, but I think we probably shouldn't - `AllowedSourceIps::Any` isn't much longer, and it's _much_ clearer.

Reply:
Good point. That was a holdover from when this was part of the bootstore, which required a default for backwards compatibility. Will remove shortly.

Reply:
Done in d72f7dc